CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # AWS AppSync SDK for iOS - CHANGELOG
 
-The AWS AppSync SDK for iOS enables you to access your AWS AppSync backend and perform operations like `Queries`, `Mutations` and `Subscriptions`. The SDK also includes support for offline operations.
+The AWS AppSync SDK for iOS enables you to access your AWS AppSync backend and perform operations like `Queries`, `Mutations` and `Subscriptions`. The SDK
+also includes support for offline operations.
```

Comment:
I think there is a line break inserted in error?

Reply:
That's intentional, I'm inserting hard line breaks into the long lines of text to make reviews & editing easier in text editors like vi.